lib/redmine_xlsx_format_issue_exporter/issues_controller_patch.rb

```diff
@@ -0,0 +1,36 @@
+require_dependency 'issues_controller'
+
+module RedmineXlsxFormatIssueExporter
+  module IssuesControllerPatch
+    include XlsxExportHelper
+
+    def query_issues
+      options = {:order => sort_clause, :offset => @offset, :limit => @limit}
+      if (Redmine::VERSION::MAJOR <= 3) && (Redmine::VERSION::MINOR <= 3) && (Redmine::VERSION::BRANCH != 'devel') then
+        options.merge!({:include => [:assigned_to, :tracker, :priority, :category, :fixed_version]})
```

Comment:
__[rubocop]__
* Performance/RedundantMerge: Use `options[:include] = [:assigned_to, :tracker, :priority, :category, :fixed_version]` instead of `options.merge!({:include => [:assigned_to, :tracker, :priority, :category, :fixed_version]})`. [:ref](https://github.com/JuanitoFatas/fast-ruby#hashmerge-vs-hash-code)

<!-- Comment by [SideCI](https://sideci.com) -->